app/controllers/application_controller.rb

```diff
@@ -2,4 +2,23 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  before_filter :configure_permitted_parameters, if: :devise_controller?
+
```

Comment:
I don't really know how this is done on rails 4 but this method seems a little bit big for the task. @alexebird  what do you think?